src/deepagents/graph.py

```diff
@@ -53,6 +56,8 @@ def create_deep_agent(
     built_in_tools = [write_todos, write_file, read_file, ls, edit_file]
     if model is None:
         model = get_default_model()
+    elif isinstance(model, str):
```

Comment:
why is this needed? i think just passing model string to create_react_agent is good enough?